classes/Util/Immutable.php

```diff
@@ -1,21 +0,0 @@
-<?php
-
-namespace OpenCFP\Util;
-
-use Exception;
-
-trait Immutable
```

Comment:
What do you think about cherry-picking cf6254927a406c3f77f1159a8ec6cfbcbbf0ef73 and consequently removing

* `Immutable`
* `ImmutableObject`

- both of which are in fact unused - as well as

* `ImmutableTest`

in a separate PR?

While I think all of the changes proposed in this PR here are good, I tend to prefer smaller PRs addressing single issues, as they are easier to review (and thus make it faster into `master`). Hope you don't mind me expressing this concern.

Reply:
Have to admit i had to figure out how to cherry pick a commit, but this is now done in #528